python/tchannel/tornado/tchannel.py

```diff
@@ -78,6 +79,9 @@ def __init__(self, hostport=None, process_name=None, known_peers=None):
         :param known_peers:
             A list of host-ports at which already known peers can be reached.
             Defaults to an empty list.
+        :param caller_name:
+            Name of the caller. This will be set in the TChannel
+            protocol header.
```

Comment:
That it will be sent out as a header is not a detail users care about. It's just an argument they have to provide that specifies the name of the service their TChannel represents.

Also, since this is basically a required header at the protocol level, it may be reasonable to make it required.